modules/search/instant-search/components/overlay.jsx

```diff
@@ -33,13 +37,15 @@ const Overlay = ( { children, closeColor, closeOverlay, colorTheme, isVisible, o
 			].join( ' ' ) }
 			style={ { opacity: opacity / 100 } }
 		>
-			<button
+			<div
```

Comment:
Changing a `div` to `button`, even when given a `button` role, decreases accessibility. Are we doing this to avoid styling issues from the theme?

Reply:
Yes, exactly. Since almost all themes include default styling for the `button` element it would be very hard for us to target it in order to remove all styling.

For example, Twenty Twenty targets it as `button:not(.toggle)` which is highly specific and nearly impossible to target, even with a narrow `button.jetpack-instant-search__overlay-close`.

To be clear though, I've transformed it into a div because that's what my skills allowed me to do. I imagine a link (`a`) would be a better fit, but it requires more wiring and functions that I could do on friday.